docs/phylo_bgs.slim

```diff
@@ -0,0 +1,30 @@
+initialize()
+{
```

Comment:
perhaps say up top what the things that need to be defined are? (`treefile`, `gens`, and `popsize`?)